src/test/resources/pages/instructorCommentsPageAddSc.html

```diff
@@ -227,7 +227,7 @@ <h4 class="col-sm-9 text-color-primary">
                       <b>
                         all students in this course
                       </b>
-                      [Fri, 01 Jun 2012, 07:20 PM UTC] (last edited by comments.instructor1@course1.tmt at ${datetime.now})
+                      [Fri, 01 Jun 2012, 07:20 PM UTC] (last edited by comments.instructor1@course1.tmt at Thu, 28 Jul 2016, 09:54 PM UTC)
```

Comment:
You need to run the tests with `-Duser.timezone=UTC` VM args, otherwise there are certain times in the day where this kind of unwanted replacement happens.